app/main.py

```diff
@@ -1,3 +1,44 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if isinstance(other, Distance):
+            return Distance(km=other.km + self.km)
+        return Distance(km=other + self.km)
+
+    def __iadd__(self, other):
+        if isinstance(other, int) or isinstance(other, float):
+            x = other
+            self.km += x
+        else:
+            x = other.km
+            self.km += x
+        return self
+
+    def __mul__(self, other):
+        return Distance(self.km * other)
+
+    def __truediv__(self, other):
+        return Distance(round(self.km / other, 2))
+
+    def __lt__(self, other):
+        return self.km < other
+
+    def __gt__(self, other):
+        return self.km > other
```

Comment:
Consider that `other` could have different types (including `Distance`, `float`, `int`)

Reply:
but tests is done?